doc/examples/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -1,383 +1,520 @@
-:moveit1:
-
-..
-   Once updated for MoveIt 2, remove all lines above title (including this comment and :moveit1: tag)
-
 MoveIt Setup Assistant
 ========================
 
-.. image:: MoveIt_setup_assistant_launch.png
+.. image:: setup_assistant_launch.png
    :width: 700px
+   :align: center
 
 Overview
-----------------------
-The MoveIt Setup Assistant is a graphical user interface for
-configuring any robot for use with MoveIt. Its primary function is
-generating a Semantic Robot Description Format (SRDF) file for your
-robot. Additionally, it generates other necessary configuration files
-for use with the MoveIt pipeline. To learn more about the SRDF, you
-can go through the :doc:`URDF/SRDF Overview </doc/examples/urdf_srdf/urdf_srdf_tutorial>`
-page.
+--------
+The MoveIt Setup Assistant is a graphical user interface for configuring any robot for use with MoveIt.
+Its primary function is generating a Semantic Robot Description Format (SRDF) file for your robot,
+which specifies additional information required by MoveIt such as planning groups, end-effectors, and various kinematic parameters.
+Additionally, it generates other necessary configuration files for use with the MoveIt pipeline.
+To use the MoveIt Setup Assistant, you will need to have a URDF file for your robot.
+
+Once you have a URDF file, you can open the MoveIt Setup Assistant and import your URDF.
+This tutorial will guide you through a series of steps to configure various aspects of your robot,
+such as defining its kinematic structure, specifying planning groups and end-effectors, and collision checking related setting.
+To learn more about the URDF and SRDF, you can refer to the `URDF SRDF Overview <https://moveit.picknik.ai/humble/doc/examples/urdf_srdf/urdf_srdf_tutorial.html#>`_ page.
 
 Getting Started
-------------------------
+---------------
 
-MoveIt and ROS
+MoveIt and ROS 2
 
-* Follow the instructions for :moveit_website:`installing MoveIt<install>`
+* Follow the `instructions for installing MoveIt <https://moveit.picknik.ai/humble/doc/tutorials/getting_started/getting_started.html>`_
```

Comment:
For all these types of URLs, instead of using the absolute links, you can do relative links like this:

```
:doc:`instructions for installing MoveIt </doc/tutorials/getting_started/getting_started>`
```

Where the link is the relative path of the `.rst` file in the repo directory.

This makes it so that when you switch doc versions, the right link is maintained instead of always pointing to Humble.

Reply:
Okay, I understand the reason now. Thank you! 

The URLs use :doc: now to refer to other files in the documentation.